src/style.css

```diff
@@ -1,3 +1,89 @@
+* {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+
+  --active-color:#00ACDC;
+}
+
+
+
 body {
   margin: 0;
 }
+
+
+header{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo_container{
+  display: flex;
+}
+
+nav{
+  position: relative;
+  height: 100%;
+}
+
+ul{
```

Comment:
<img width="1059" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/7d727d8f-ae68-4fc8-a30f-d65d420c01e2">

Reply:
The same in other places